:scripts/hw4_solution 29-35.r

```diff
@@ -0,0 +1,128 @@
+# Assignment 1:  
+library(tweedie) 
+library(ggplot2)
+
+simTweedieTest <-  
+  function(N){ 
+    t.test( 
+      rtweedie(N, mu=10000, phi=100, power=1.9), 
+      mu=10000 
+    )$p.value 
+  } 
+
+
+# Assignment 2:  
+MTweedieTests <-  
+  function(N,M,sig){ 
+    sum(replicate(M,simTweedieTest(N)) < sig)/M 
+  } 
+
+
+# Assignment 3:  
+df <-  
+  expand.grid( 
+    N = c(10,100,1000,5000, 10000), 
+    M = 1000, 
+    share_reject = NA) 
+
+
+library(purrr)
+
+
+df$share_reject <- pmap(list(N = df$N, M = df$M, sig = .05), MTweedieTests)
```

Comment:
I am not sure why this is not faster, as I am not very familiar with the pmap library. The downside with using libraries like this is that it is hard to know what actually happens when you run the function.